react/features/base/responsive-ui/actions.ts

```diff
@@ -59,9 +59,10 @@ export function clientResized(clientWidth: number, clientHeight: number) {
             dispatch({
                 type: CLIENT_RESIZED,
                 clientHeight,
-                clientWidth: availableWidth
+                clientWidth,
+                videoSpaceWidth: availableWidth
             });
-            dispatch(setAspectRatio(clientWidth, clientHeight));
+            dispatch(setAspectRatio(availableWidth, clientHeight));
```

Comment:
This could create a problem no? When the remaining space is too little and may morph into the "mobile" view.

Since the panels are part of that UI on web but not on mobile web I thinjk we should not use the availableWidth but rather clientWidth when computing the aspect ratio.

Reply:
@saghul this PR is just renaming stuff. Not sure if any problem exists. But even if it exist it is not part of this PR and can be resolved as separate task